### ISSUE TITLE
Default Compendium Setting?

### ISSUE THREAD

**Issue by sirkerry** [Forelius/fantastic-depths#37]
Would it be possible to have a game configuration setting that disables the default fantastic-depths compendium and allows a different default compendium to be specified?

**Comment by Forelius** [comment]
Any compendium can be used, as long as it is compatible with the Fantastic Depths system. There is no default per se.

**Comment by sirkerry** [comment]
Yes, I'm aware that any compendium can be used, what I want really is the ability to suppress the fantastic-depths compendium from loading at all.

**Comment by Forelius** [comment]
I don't have a way to do that currently. The compendiums/packs are defined in the system file, which is immutable. Perhaps if there is enough interest I can move the compendiums to a separate module.

**Comment by Forelius** [comment]
I'll be separating the compendium from the game system.
https://github.com/Forelius/fade-compendiums
